pkg/grpc/universalapi/api_workflow.go

```diff
@@ -0,0 +1,195 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package universalapi
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/dapr/components-contrib/workflows"
+	"github.com/dapr/dapr/pkg/messages"
+	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
+)
+
+func (a *UniversalAPI) GetWorkflowAlpha1(ctx context.Context, in *runtimev1pb.GetWorkflowRequest) (*runtimev1pb.GetWorkflowResponse, error) {
+	if in.InstanceId == "" {
+		err := messages.ErrMissingOrEmptyInstance
+		a.Logger.Debug(err)
+		return &runtimev1pb.GetWorkflowResponse{}, err
+	}
+	if in.WorkflowComponent == "" {
+		err := messages.ErrNoOrMissingWorkflowComponent
+		a.Logger.Debug(err)
+		return &runtimev1pb.GetWorkflowResponse{}, err
+	}
+	workflowComponent := a.WorkflowComponents[in.WorkflowComponent]
+	if workflowComponent == nil {
+		err := messages.ErrWorkflowComponentDoesNotExist.WithFormat(in.WorkflowComponent)
+		a.Logger.Debug(err)
+		return &runtimev1pb.GetWorkflowResponse{}, err
+	}
+	req := workflows.WorkflowReference{
+		InstanceID: in.InstanceId,
+	}
+	response, err := workflowComponent.Get(ctx, &req)
+	if err != nil {
+		innerErr := messages.ErrWorkflowGetResponse.WithFormat(in.InstanceId, err)
+		a.Logger.Debug(innerErr)
+		return &runtimev1pb.GetWorkflowResponse{}, innerErr
+	}
+
+	id := &runtimev1pb.WorkflowReference{
+		InstanceId: response.WFInfo.InstanceID,
+	}
+
+	t, err := time.Parse(time.RFC3339, response.StartTime)
+	if err != nil {
+		innerErr := messages.ErrTimerParse.WithFormat(err)
+		a.Logger.Debug(innerErr)
+		return &runtimev1pb.GetWorkflowResponse{}, innerErr
+	}
+
+	res := &runtimev1pb.GetWorkflowResponse{
+		InstanceId: id.InstanceId,
+		StartTime:  t.Unix(),
```

Comment:
This is unrelated from this PR but I don't think we use UNIX timestamps anywhere else in Dapr. Consider (for a v1beta1, because this a breaking change) switching to printing the time as RFC3389
CC: @cgillum

Reply:
@ItalyPaleAle how do you feel about using `google.protobuf.Timestamp` for timestamps? This is what we use in the `durabletask-go` proto definitions and it removes the need for us to worry about string format consistency or otherwise do any interpretation in the generated client/server code.

Reply:
Issue to track the request https://github.com/dapr/dapr/issues/6098